docs/version_bump_template.md

```diff
@@ -60,14 +59,15 @@ Available flags to create your tests with some data examples:
 - $ -testCase "TestLocalPathProvisionerStorage"
 - $ -workloadName "bandwidth-annotations.yaml"
 - $ -description "Description of your test"
-- $ -debug true
 ```
 
 * All non-boolean arguments are comma separated in case you need to send more than 1.
 
 * If you need to separate another command to run as a single here, separate those with " : " as this example:
+That could separators like ";" , "|" , "&&" etc.
```

Comment:
Suggestion: `The shell command separators can also be used within the commands like ";" , "|" , "&&" etc.`